packages/theme/src/cli/utilities/theme-environment/remote-theme-watcher.ts

```diff
@@ -25,12 +24,12 @@ export async function reconcileAndPollThemeEditorChanges(
   },
 ) {
   outputDebug('Initiating theme asset reconciliation process')
+  await localThemeFileSystem.ready()
```

Comment:
this ensures that our themeFileSystem is fully mounted before we started initiating reconciliation